README.md

```diff
@@ -1,13 +1,31 @@
-<img height="256" align="left" src="https://user-images.githubusercontent.com/39812919/122685073-b5075180-d1df-11eb-8172-9ee6ada53190.gif" />
+## 👩 Hi!
 
-#### about me
+Eu sou Isabella Basso
+ 
+Redpill 💊 + Desenvolvedora
 
-- learning C + django
-- software engineer @ Red Hat Plumbers team
-- undergrad student @ [universidade de são paulo](https://www5.usp.br)
-- proud member of [lkcamp.dev](LKCamp)
+Cursando Ciências Moleculares 🧪 - USP.
 
-#### what i do
+Meu foco atualmente está em conseguir um blahaj 🦈 e em segundo plano em fazer bugs baixo nível com C (odeio Rust btw 🦀🔫)
 
-- now: C static analysis tool + linux kernel unit testing
-- previous: whatsapp bot + AR for the web
+<div>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=isinyaaa&show_icons=true&theme=tokyonight"/>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=isinyaaa&layout=compact&theme=tokyonight"/>
+</div>
+
+### ❤️ Let's get connected:
+
+[![Discord](https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/im-not-gonna-put-your-real-discord-lol-im-not-that-evil)
+[![Linkdin](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/isabellabasso/)
+[![Twitter](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/isinyaaa)
+
+### 🛠️ Technologies e Tools I use:
+
+<div>
+<img align="center" alt="Isabella-c" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/c/c-original.svg"/>
+<img align="center" alt="Isabella-python" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg"/>
+<img align="center" alt="Isabella-django" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/django/django-original.svg"/>
+<img align="center" alt="Isabella-linux" height="30" width="40" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linux/linux-original.svg"/>
```

Comment:
Isn't this list missing on some key technologies? I.e. firebase, latex... Have a look at [isinyaaa.github.io/](https://isinyaaa.github.io/CV.pdf)